src/utils/githubDataUtil.js

```diff
@@ -53,3 +63,16 @@ export const parseRepoData = (repoData) => {
     },
   );
 };
+
+/**
+ * takes a list of branches and puts the main branch at the top
+ * @param {Array<String>} branches
+ * @param {String} mainBranch
+ */
+export const orderRepoBranches = (branches, mainBranch) => {
+  const copy = branches.slice();
+  const defaultBranchIndex = copy.indexOf(mainBranch);
```

Comment:
Is the name mainBranch confusing? Is that the "main" branch or the primary branch?

Reply:
renamed to defaultbranch, its called that in github api